airbyte-integrations/connectors/destination-mssql/src/test-integration/java/io/airbyte/integrations/destination/mssql/MSSQLIntegrationTest.java

```diff
@@ -0,0 +1,193 @@
+import io.airbyte.db.Database;
+import io.airbyte.db.Databases;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.destination.ExtendedNameTransformer;
+import io.airbyte.integrations.standardtest.destination.TestDestination;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.jooq.JSONFormat;
+import org.jooq.JSONFormat.RecordFormat;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.containers.MSSQLServerContainer;
+
+public class MSSQLIntegrationTest extends TestDestination {
+
+  private static final JSONFormat JSON_FORMAT = new JSONFormat().recordFormat(RecordFormat.OBJECT);
+
+  private static MSSQLServerContainer<?> db;
+  private ExtendedNameTransformer namingResolver = new ExtendedNameTransformer();
+  private JsonNode configWithoutDbName;
+  private JsonNode config;
+
+  @Override
+  protected String getImageName() {
+    return "airbyte/destination-mssql:dev";
+  }
+
+  private JsonNode getConfig(MSSQLServerContainer<?> db) {
+    return Jsons.jsonNode(ImmutableMap.builder()
+        .put("host", db.getHost())
+        .put("port", db.getFirstMappedPort())
+        .put("username", db.getUsername())
+        .put("password", db.getPassword())
+        .put("schema", "testSchema")
+        .build());
+  }
+
+  @Override
+  protected JsonNode getConfig() {
+    return getConfig(db);
+  }
+
+  @Override
+  protected JsonNode getFailCheckConfig() {
+    return Jsons.jsonNode(ImmutableMap.builder()
+        .put("host", db.getHost())
+        .put("username", db.getUsername())
+        .put("password", "wrong password")
+        .put("schema", "public")
+        .put("port", db.getFirstMappedPort())
+        .put("ssl", false)
+        .build());
+  }
+
+  @Override
+  protected List<JsonNode> retrieveRecords(TestDestinationEnv env, String streamName, String namespace) throws Exception {
+    return retrieveRecordsFromTable(namingResolver.getRawTableName(streamName), namespace)
+        .stream()
+        .map(r -> Jsons.deserialize(r.get(JavaBaseConstants.COLUMN_NAME_DATA).asText()))
+        .collect(Collectors.toList());
+  }
+
+  @Override
+  protected boolean implementsBasicNormalization() {
+    return false;
+  }
+
+  @Override
+  protected boolean implementsNamespaces() {
+    return true;
+  }
+
+  @Override
+  protected List<JsonNode> retrieveNormalizedRecords(TestDestinationEnv env, String streamName, String namespace)
+      throws Exception {
+    String tableName = namingResolver.getIdentifier(streamName);
+    // Temporarily disabling the behavior of the ExtendedNameTransformer, see (issue #1785) so we don't
+    // use quoted names
+    // if (!tableName.startsWith("\"")) {
+    // // Currently, Normalization always quote tables identifiers
+    // //tableName = "\"" + tableName + "\"";
+    // }
```

Comment:
```suggestion
    // Temporarily disabling the behavior of the ExtendedNameTransformer, see (issue #1785) so we don't
    // use quoted names
    // if (!tableName.startsWith("\"")) {
    // // Currently, Normalization always quote tables identifiers
    // //tableName = "\"" + tableName + "\"";
    // }
```

Reply:
What is the proposed change here?  The before and after appear identical.